bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -84,8 +84,11 @@ public Builder withScan(Scan scan) {
      * @return The {@link CloudBigtableScanConfiguration.Builder} for chaining convenience.
      */
     public Builder withRequest(ReadRowsRequest request) {
+      return withRequest(staticProvider(request));
+    }
+
+    Builder withRequest(ValueProvider<ReadRowsRequest> request) {
```

Comment:
How is this meant to be used? is this meant to only be used by the sequencefile pipeline? Or will there be a follow up PR to expose this to the general user?

Reply:
There absolutely needs to be further discussion about how to move this forward.  For now, it's meant to be a minimal set of changes required to use any type of `ValueProvider`s.